packages/web-react/src/components/TextField/README.md

```diff
@@ -26,7 +26,7 @@ hidden or show if the input is required.
 | `name`              | string                                                        | Input name                                                              |
 | `placeholder`       | string                                                        | Input placeholder                                                       |
 | `type`              | `email`, `number`, `password`, `search`, `tel`, `text`, `url` | Input type                                                              |
-| `validationState`   | `success`, `warning`, `error`                                 | Type of validation state                                                |
+| `validationState`   | `success`, `warning`, `danger`, `error`                       | Type of validation state                                                |
```

Comment:
link to docs/dictionaries.md and deprecation info please

Reply:
done